pubspec.yaml

```diff
@@ -1,11 +1,11 @@
 name: flutter_app_badger
 description: Plugin to update the app badge on the launcher (both for Android, iOS and macOS)
-version: 1.5.1
+version: 1.5.2
 homepage: https://github.com/g123k/flutter_app_badger
 
 environment:
-  sdk: '>=2.19.6 <4.0.0'
-  flutter: ">=3.7.12"
+  sdk: ">=3.3.0 <4.0.0"
```

Comment:
Since this is a third party plugin,, changing the minimum SDKs is not recommended.

The goal of forks is to later be migrated to the main repository.